cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -85,7 +85,8 @@ def test_provision_cloud_init(appliance, request, setup_provider, provider, prov
     provision_request = provider.appliance.collections.requests.instantiate(vm_name,
                                                                    partial_check=True)
     provision_request.wait_for_request()
-    connect_ip = mgmt_system.get_vm(vm_name).ip
+    wait_for(lambda: instance.ip_address is not None, num_sec=60)
+    connect_ip = instance.ip_address
```

Comment:
For future use - wait_for returns a tuple of (func(), time_waited), so you can capture that and at the end (when it did NOT timeout) you'll have the result of the func you passed. Saves having to call the property again.